intl/msg_hash_val.c

```diff
@@ -0,0 +1,2457 @@
+/*  RetroArch - A frontend for libretro.
+ *  Copyright (C) 2011-2017 - Daniel De Matteis
+ *  Copyright (C) 2016-2019 - Brad Parker
+ *
+ *  RetroArch is free software: you can redistribute it and/or modify it under the terms
+ *  of the GNU General Public License as published by the Free Software Found-
+ *  ation, either version 3 of the License, or (at your option) any later version.
+ *
+ *  RetroArch is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ *  PURPOSE.  See the GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with RetroArch.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ */
+
```

Comment:
The .c file for a new language shouldn't be this long I guess

Reply:
I'm sorry, I don't understand. What do you want me to change here?

Reply:
Check this file https://github.com/libretro/RetroArch/blob/master/intl/msg_hash_ru.c

I think the .c file should be as short as possible, especially when strings are not translated.

In future, .c files should not contain translatable strings anymore, because they are not synchronized with Crowdin.

Reply:
> Check this file

Okay, I get it now.

> In future, .c files should not contain translatable strings anymore, because they are not synchronized with Crowdin.

Right, I haven't told you my plan yet. Once [Ryunams PR](https://github.com/libretro/RetroArch/pull/12811) is completed, I intend to refactor this whole operation and reduce all the current .c files into just one file and have all those strings be synchronized with Crowdin.